webapp/src/services/near-nep21-util.js

```diff
@@ -83,6 +83,11 @@ export async function gasCheck() {
 }
 
 export function convertToE24Base( str ) {
+
+  let result = (str +"").padStart(24,"0")
+  result = result.slice(0,-24)+"."+result.slice(-24)
+  return result
+/*
```

Comment:
what's wrong with the code below?

Did you run tests? There are tests for this function and it works.